### DISCUSSION TITLE
How to use tabbed code blocks

### DISCUSSION THREAD

**Discussion by aidan-gallagher** [mmistakes/minimal-mistakes#3160] [Q&A]
I would like to use tabbed code blocks in my blog (https://rdmd.readme.io/docs/code-blocks#tabbed-code-blocks )

I don't think Jeykll's markdown engine (kramdown) supports it. 

Any ideas for the best solution to use tabbed code blocks?

**Comment by mmistakes** [comment]
I think you're in Jekyll plugin territory here... or potentially a custom `_include` to do it.
Either way it's not something you're going to find in core Jekyll or this theme.

**Comment by qiujames** [comment]
Decided to implement tabbed code blocks as a custom _include.
See it here https://qiujames.github.io/tabbed-content-in-mm/#demo.
Feel free to poke in the repository to take a look at the code.
It's styled similarly to the masthead and is included similarly to how galleries work.

Could have issues since I haven't thoroughly tested it yet, but it seems to be pretty clean so far.

1. Add into your _includes/ directory the following [tablist](https://github.com/qiujames/qiujames.github.io/blob/main/_includes/tablist)
2. Add into your _sass/minimal-mistakes/ directory the following [_tablist.scss](https://github.com/qiujames/qiujames.github.io/blob/main/_sass/minimal-mistakes/_tablist.scss)
3. Update your _sass/minimal-mistakes.scss to contain the import
4. Add into assets/js/ the following [tablist-script.js](https://github.com/qiujames/qiujames.github.io/blob/main/assets/js/tablist-script.js)
5. Load the js file into your site by following the typical MM [workflow](https://mmistakes.github.io/minimal-mistakes/docs/javascript/#customizing) by adding it to _config.yml or in the the `<head>`
